server.py

```diff
@@ -38,14 +38,32 @@ def index():
 def list_products():
     category        = request.args.get('category')
     discontinued    = str2bool(request.args.get('discontinued'))
-    results     = []
+    min_Price       = request.args.get('minPrice')
```

Comment:
In Slide "#25" of the "04-Intro-REST-Python-Flask.pdf" presentation, we are advised to use hyphens (e.g. 'min-price' and 'max-price') over camel-case (e.g. 'minPrice' and 'maxPrice').

I wonder if we could get away with  just using 'min' and 'max', or if this introduces confusion as it's not clear whether 'min' and 'max' refer to price or to something else (e.g. the number of elements returned). Perhaps 'min-price' and 'max-price' are better conventions - for this reason.